NewForce-at-Mountwest/nutshell-team-stumbletoe
# Added chat feature

Created chat feature.

To test: 
1) log in
2) scroll down to chat box
3) type in a message into the bar at the bottom
4) hit send
5) you should see a new message in the main display area
